libminifi/src/core/repository/FileSystemRepository.cpp

```diff
@@ -17,19 +17,22 @@
  */
 
 #include "core/repository/FileSystemRepository.h"
-#include <string>
+
 #include <filesystem>
+#include <string>
+
+#include "core/ForwardingContentSession.h"
 #include "io/FileStream.h"
+#include "utils/Locations.h"
 #include "utils/file/FileUtils.h"
-#include "core/ForwardingContentSession.h"
 
 namespace org::apache::nifi::minifi::core::repository {
 
 bool FileSystemRepository::initialize(const std::shared_ptr<Configure>& configuration) {
   if (std::string directory_str; configuration->get(Configure::nifi_dbcontent_repository_directory_default, directory_str) && !directory_str.empty()) {
     directory_ = directory_str;
   } else {
-    directory_ = configuration->getHome().string();
+    directory_ = utils::getMinifiDir();
```

Comment:
On Windows, this needs a `.string()` as before. Or (better) `directory_` could be changed from a `string` to a `path`, but that may snowball into too many changes.

Reply:
thanks 👍 looked into it and it does look like snowbally so I will not burden the PR with that